spm/.htaccess

```diff
@@ -0,0 +1,24 @@
+# Allow all (*) origins to access the resource on the server
+Header set Access-Control-Allow-Origin *
+
+# Allow following symbolic links / symlinks / softlinks
+Options +FollowSymLinks
+
+# Turn off MultiViews
+Options -MultiViews
+
+RewriteEngine On
+
+## Redirects 
+# DOCS
+RewriteRule ^(.*)docs/colpadef https://userpages.uni-koblenz.de/~process-science/docs/fg-bks-social-process-mining-collaboration-detection/main/ [R=303,L]
```

Comment:
have you tested this rule? you are not adding the `$` at the end, which looks odd. Also, the `(.*)` before the docs redirection.

Reply:
Hey @dgarijo, thank you for the reviews. Yes, these redirects work. However, I will add the „$“ in a new commit!